src/pages/home/IncidentCard.tsx

```diff
@@ -1,9 +1,10 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Incident as IncidentInterface } from '../../api'
 import IncidentUpdate from './IncidentUpdate'
 import styled from 'styled-components/macro'
 import { Link } from 'react-router-dom'
 import { LinkButton } from '../../components/Buttons'
+import { Collapse } from 'react-collapse'
```

Comment:
BTW there are no changes to `package.json` and `package-lock.json` so we have no way to install this module. You should have run `npm i react-collapse` and commited the files

Reply:
ouch! My bad. I'll fix it